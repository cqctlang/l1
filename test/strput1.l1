s = "abc";
print(s);
strput(s, 1, "de");
print(s);
s = "abc";
strput(s, 2, "e");
print(s);
s = "abc";
strput(s, 0, "");
print(s);
s = "abc";
strput(s, 0, "def");
print(s);
