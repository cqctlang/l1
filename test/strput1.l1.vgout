entry: 0 params, 0 locals, 0 temps
abc
ade
abe
abc
def
