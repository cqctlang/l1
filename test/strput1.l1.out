entry: var params, 1 locals, 0 temps
"abc"
"ade"
"abe"
"abc"
"def"
